news/8327.bugfix

```diff
@@ -0,0 +1 @@
+The pip debug command takes module versions from vendor.txt even when they are debundled.  Enhance the debug command output to instead show the location, name, and version of the debundled modules when appropriate.
```

Comment:
I would name this with `.vendor`, so when we generate news it gets more exposure to the people repackaging pip.